src/overlays/actors/ovl_En_Zl3/z_en_zl3.c

```diff
@@ -1926,10 +1926,10 @@ s32 func_80B57890(EnZl3* this, GlobalContext* globalCtx) {
 
 void func_80B57A74(GlobalContext* globalCtx) {
     Actor* actorIt = globalCtx->actorCtx.actorLists[ACTORCAT_PROP].head;
-
+    //! @bug checks for encount2 in ACTORCAT_PROP but encount2 is in ACTORCAT_ENEMY so this condition is never met
```

Comment:
En_Encount2 has a category change to `ACTORCAT_PROP` in one of the init cases (through `Actor_ChangeCategory`), so this is probably not a bug?

Reply:
where is this category change? i dont see it in encount2

Reply:
Nvm, I think I was looking at `En_Fire_Rock` (the actor spawned) and got confused because both actors are in these changes. In fact, there is a good chance this bug was caused by a similar confusion 😅

Reply:
agreed yea, thats prob what happened lol